spec/fortnox/api/repositories/unit_spec.rb

```diff
@@ -19,16 +19,16 @@
   # When recording new VCR cassettes, code needs to be changed to a unique value
   include_examples '.save',
                    :description,
-                   additional_attrs: { code: 'blarg5' }
+                   additional_attrs: { code: 'blarg7' }
```

Comment:
... Could we in future use reasonable strings for test data? Run a lorem word if you can't think of something appropriate 🙂

Reply:
Haha, ye sure 😄